packages/client-config/src/client-config-contributor/auth_client_config_contributor.ts

```diff
@@ -87,9 +87,11 @@ export class AuthClientConfigContributor implements ClientConfigContributor {
       );
     }
 
-    if (authOutput.payload.oauthDomain) {
+    if (authOutput.payload.oauthClientId) {
```

Comment:
This is the fix.
oauthDomain would only be set if domainPrefix was set even if oauth is enabled, but oauthClientId is always set if oauth is enabled. 
Checking for oauthClientId is a sure way to determine if oauth settings should be output to the config.